app/src/main/java/org/mifos/selfserviceapp/ui/activities/HomeActivity.java

```diff
@@ -76,7 +75,7 @@ public void onCreate(Bundle savedInstanceState) {
 
         setToolbarTitle(getString(R.string.home));
 
-        clientId = getIntent().getExtras().getLong(Constants.CLIENT_ID);
+        clientId = preferencesHelper.getClientId();
```

Comment:
@dilpreet96 Good work 👍, This is the correct way of doing this, During application period everyone was contributing so I did not get the chance to fix this. Please if you find this anywhere in the project do same.

Reply:
test